### DISCUSSION TITLE
Is it possible to reset the credentials of a local PockeBase instance?

### DISCUSSION THREAD

**Discussion by delliottg** [pocketbase/pocketbase#4496] [Q&A] [answered]
@ganigeorgiev, @cap'n #0[|<[-7 |>0$7 I tried to restore a backup from PocketHost to a local PockeBase instance.  Once I started the restore, I was prompted to log in again.  When I attempted to log in using my credentials I'd saved in Last Pass, it gave me an "Invalid Login Credentials" error.  So I tried to do a password reset, but since I'd setup my mail with MailTrap, the email never went anywhere and there's no record of it in MailTrap.  According to Cap'n on the Discord channel, it's possible to sync the passwords between the two instances from the PocketBase UI, which I can no longer get to.

Is it possible to force a new instance or reset the password on the existing one?  I don't have anything in the local instance so I don't care if I nuke it.

**Comment by ganigeorgiev** [comment]
I'm having difficulties understanding the description but if you want to reset your admin password, you could try running:

```sh
./pocketbase admin update test@example.com yourNewPassword

# or create a new admin if you don't remember your email
./pocketbase admin create new@example.com yourPassword
```

**Reply by ganigeorgiev** [reply]
> ... I don't care if I nuke it

If you want to start from scratch, you can delete your existing `pb_data`.

**Reply by delliottg** [reply]
The first suggestion didn't work, it didn't like my password giving me a error of:

fish: No matches for wildcard '9xhz%*Sn^2HJF1%F'.  See help for wildcard globbing.

I'm going to try your second suggestion.

**Reply by ganigeorgiev** [reply, marked as answer]
It should work if you wrap your password in single quotes to avoid the terminal wrongly interpreting the special characters.
